frontend/src/pages/multisig/PageMultisig.js

```diff
@@ -131,26 +131,12 @@ export default function PageMultisig() {
         {multisigAccounts?.status !== "pending" && !accounts?.length ? (
           <Box
             sx={{
-              mt: 6,
+              mt: 2,
             }}
           >
-            <Typography variant="h6" fontWeight={600}>
+            <Typography variant="h6">
               No Multisig accounts found on your address
             </Typography>
-            <Button
```

Comment:
We also talked about removing the 'Create New Multisig' tab and adding a new button at the end.

Reply:
Means instead of `Create New Multisig` we have create button with `New Multisig +` ?